packages/loader/container-loader/src/container.ts

```diff
@@ -330,16 +335,21 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     public static async rehydrateDetachedFromSnapshot(
         loader: Loader,
         snapshot: string,
+        protocolDetails?: IProtocolDetails,
     ): Promise<Container> {
         const container = new Container(
             loader,
             {});
+
```

Comment:
I wonder - should we instead pass protocolDetails as constructor argument? And thus not need to change various pass-through functions?

Reply:
will do.